ansible-ipi-install/roles/bootstrap/files/Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM docker.io/centos:centos7
+FROM centos:8
 ARG foreman_url=https://foreman.example.com
 RUN yum install -y epel-release \
-    https://yum.theforeman.org/releases/2.1/el7/x86_64/foreman-release.rpm \
-    centos-release-scl-rh \
-    rh-ruby25-ruby
+        python3
 
-RUN yum install -y tfm-rubygem-hammer_cli \
-    tfm-rubygem-hammer_cli_foreman
+RUN yum install -y https://yum.theforeman.org/releases/2.1/el8/x86_64/foreman-release.rpm
```

Comment:
Lets condense into one RUN/yum command, would result in fewer layers in the image.

Reply:
Move to dnf as it is centos8